artifacts/definitions/Server/Enrichment/Virustotal.yaml

```diff
@@ -0,0 +1,40 @@
+name: Server.Enrichment.Virustotal
+author: Wes Lambert -- @therealwlambert
+description: | 
+  Submit a file hash to Virustotal for details. Default Public API restriction is 4 requests/min.
+  
+  This artifact can be called from within another artifact (such as one looking for files) to enrich the data made available by that artifact.
+  
+  Ex.
+  
+    `SELECT * from Artifact.Server.Enrich.Virustotal(Hash=$YOURHASH)`
+  
+  `TO-DO`: Implement a timer to spread out requests
+  
+type: SERVER
+
+parameters:
+    - name: Hash
+      type: string
+      description: The file hash to submit to Hybrid Analysis (MD5, SHA1, SHA256).
+      default: 
+      
+    - name: VirustotalKey
+      type: string
+      description: API key for Hybrid Analysis. Leave blank here if using server metadata store.
+      default:
+
+sources:
+  - queries:
+    - |
+        LET Creds = if(
+           condition=VirustotalKey,
+           then=VirustotalKey,
+           else=server_metadata().VirustotalKey)
+    - |
+        LET URL <= 'https://www.virustotal.com/api/v3/files/' + Hash
+    - | 
+        LET Data = SELECT parse_json(data=Content) AS VTData from http_client(url=URL, headers=dict(`x-apikey`=Creds))
+    - |
+        SELECT + format(format='%v/%v', args=[VTData.data.attributes.last_analysis_stats.malicious, VTData.data.attributes.last_analysis_stats.malicious + VTData.data.attributes.last_analysis_stats.undetected]) As VTRating, 
```

Comment:
I fixed the description for the VT key if that's what you were referring to -- otherwise, are you referring to VTRating?  I made it so it is formatted the rating as "x  of y" (engines rated this as bad) , or as "x/y".  Might not be the best approach, so open to your thoughts there.

Reply:
Oh no i was referring to the spurious + there. I thought it was a syntax error but apparently the VQL engine seems happy with it.

Reply:
Oh! 😆 . Yeah, I didn't notice that until you just now pointed it out -- assume it just ignored it? I can fix that real quick.